src/QSymbolicsBase/basic_superops.jl

```diff
@@ -0,0 +1,64 @@
+##
+# Superoperator representations
+##
+
+"""Kraus representation of a quantum channel
+
+```jldoctest
+julia> @superop ℰ;
+
+julia> @op A₁; @op A₂; @op A₃;
+
+julia> K = kraus(ℰ, A₁, A₂, A₃);
+
+julia> @op ρ;
+
+julia> K*ρ
+(A₁ρA₁†+A₂ρA₂†+A₃ρA₃†)
+```
+"""
+@withmetadata struct KrausRepr <: Symbolic{AbstractSuperOperator}
+    sop
+    krausops
+end
+isexpr(::KrausRepr) = true
+iscall(::KrausRepr) = true
+arguments(x::KrausRepr) = [x.sop, x.krausops]
+operation(x::KrausRepr) = kraus
+head(x::KrausRepr) = :kraus
+children(x::KrausRepr) = [:kraus, x.sop, x.krausops]
+kraus(s::Symbolic{AbstractSuperOperator}, xs::Symbolic{AbstractOperator}...) = KrausRepr(s,collect(xs))
+symbollabel(x::KrausRepr) = symbollabel(x.sop)
+basis(x::KrausRepr) = basis(x.sop)
+Base.show(io::IO, x::KrausRepr) = print(io, symbollabel(x))
```

Comment:
and this will need to be changed, maybe to something like `𝒦[op1, op2, op3, ...]` or something else if there is a popular notation for Kraus maps.